src/components/MemberArea/model.js

```diff
@@ -98,6 +100,7 @@ export default {
         value: 'linkedin',
         label: 'LinkedIn',
         prefix: 'https://linkedin.com/in/',
+        validate: value => !!value && linkedinValidation(value),
```

Comment:
I think..
```suggestion
        validate: value => linkedinValidation(value),
```
Because it's not mandatory so it's valid if it's empty **or** it matchs the pattern.

Reply:
Yeah I agree 👍 💯